examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -119,10 +137,9 @@ typedef bmpl::vector<
     particlePusher<UsedParticlePusher>,
     shape<UsedParticleShape>,
     interpolation<UsedField2Particle>,
-    current<UsedParticleCurrentSolver>
-    #if(PARAM_IONIZATION==1)
-    ,ionizer<particles::ionization::BSI<PIC_Electrons> >
-    #endif
+    current<UsedParticleCurrentSolver>, 
+    atomicNumbers<Helium>,
+    ionizer<particles::ionization::BSI<PIC_Electrons> >
```

Comment:
Ah, I just still had that in mind somehow and didn't look at the file path. My bad .